src/cli_instance.rs

```diff
@@ -153,6 +157,31 @@ impl CliInstance {
                 .expect("failed to set vsock socket path");
         }
 
+        if !args.host_device.hostdev_id.is_none() {
```

Comment:
```suggestion
        if let Some(hostdev_id) = args.host_device.hostdev_id {
```

Reply:
actually this is for checking whether user provide hostdev_id and bus_slot_func in the command line (which are the must parameters) and I was being lazy to just check one of the parameters......

I have completed this part and also added a comment above.